src/util/CurrencyWalletHelpers.js

```diff
@@ -1,7 +1,9 @@
 // @flow
 
+import { bns } from 'biggystring'
```

Comment:
`bns` is not exported by biggystring: https://github.com/EdgeApp/biggystring/blob/master/src/index.ts#L383 

Use `* as bns` or `{ sub }`.

Reply:
Are you sure about it? ;)

<img width="634" alt="Screen Shot 2022-02-16 at 11 05 44" src="https://user-images.githubusercontent.com/4012041/154338204-6914d9f4-0891-4642-b3c2-bf26f3e22b2d.png">